src/agent/centralized_configuration/src/centralized_configuration.cpp

```diff
@@ -37,7 +42,7 @@ namespace centralized_configuration
                         module_command::Status::FAILURE, "CentralizedConfiguration group set failed, no function set"};
                 }
             }
-            else if (commnandAsJson["command"] == "update-group")
+            else if (command == "update-group")
```

Comment:
We should also log something if the command is not recognized.

Reply:
There are no log messages anywhere in the component, we could create an issue to review where they need to be added or do it as part of the following tasks. What do you think?